Gopkg.lock

```diff
@@ -845,8 +846,8 @@
     "storage/v1beta1",
   ]
   pruneopts = "UT"
-  revision = "74b699b93c15473932b89e3d1818ba8282f3b5ab"
-  version = "kubernetes-1.13.3"
+  revision = "05914d821849570fba9eacfb29466f2d8d3cd229"
+  version = "kubernetes-1.13.1"
```

Comment:
lol

Reply:
ugh! really not sure whats happening, may be cleaning up local folders helped, glad its back to normal.